app/cdap/components/Cloud/Profiles/DetailView/Content/ProfileAssociations/index.js

```diff
@@ -28,6 +28,7 @@ import {
   getNodeHours,
 } from 'components/Cloud/Profiles/Store/ActionCreator';
 import { Observable } from 'rxjs/Observable';
+import { createReplicatorDetailUrl } from '../../../../../Replicator/utilities';
```

Comment:
Can this be an absolute path?

Reply:
ahh yeah i just used the built in vs code import and it did this :/